README.md

```diff
@@ -45,3 +49,6 @@ for more details.
 [cpplint]: https://github.com/google/styleguide/tree/gh-pages/cpplint
 [emacs]: https://raw.githubusercontent.com/google/styleguide/gh-pages/google-c-style.el
 [xml]: https://google.github.io/styleguide/xmlstyle.html
+[dart]: https://www.dartlang.org/guides/language/effective-dart
+[go]: https://github.com/golang/go/wiki/Style
```

Comment:
It looks like this list of links is ordered according to the sequence that things appear in the prose, but you inserted [go] before [dart] up there, and after it here.